src/System.Runtime/tests/System/TypeTests.netcoreapp.cs

```diff
@@ -36,5 +36,42 @@ public void IsSZArray(Type type, bool expected)
         {
             Assert.Equal(expected, type.IsSZArray);
         }
+
+        private static IEnumerable<object[]> VariableBoundArrayOrNotTypes()
+        {
+            // Not arrays
+            yield return new object[] { typeof(void), false };
+            yield return new object[] { typeof(int), false };
+            yield return new object[] { typeof(int[]).MakeByRefType(), false };
+            yield return new object[] { typeof(TypeTests), false };
+            yield return new object[] { Type.GetType("System.Int32"), false };
+            yield return new object[] { typeof(Outside<int>.Inside<string>), false };
+
+            // Arrays, but SZ arrays
+            yield return new object[] { Type.GetType("System.Int32[]"), false };
+            yield return new object[] { typeof(int[]), false };
```

Comment:
Yes, let's get rid of the duplicate types. (If we want to test equivalence, that's better done in its own test that compares the Types directly.)

We don't need the Array.CreateInstance(), Type.GetType() or MakeArrayType() (other than the MakeArrayType(1) case.) variants. Some of these will cause extra problems on .NET Native runs.